Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -1294,6 +1298,34 @@ namespace Multiplayer
         }
     }
 
+    void MultiplayerSystemComponent::OnRootSpawnableReady([[maybe_unused]] AZ::Data::Asset<AzFramework::Spawnable> rootSpawnable, [[maybe_unused]] uint32_t generation)
+    {
+        SpawnPlayersWaitingToBeSpawned();
+    }
+
+    void MultiplayerSystemComponent::SpawnPlayersWaitingToBeSpawned()
+    {
+        IMultiplayerSpawner* spawner = AZ::Interface<IMultiplayerSpawner>::Get();
+        if (!spawner)
+        {
+            AZLOG_ERROR("Attempting to spawn players on level load failed. No IMultiplayerSpawner found. Ensure one is registered.");
+            return;
+        }
+
+        for (const auto& [id, datum] : m_playersWaitingToBeSpawned)
+        {
+            NetworkEntityHandle controlledEntity = spawner->OnPlayerJoin(id, datum);
+
+            // If there wasn't any spawner available, wait until a level loads and check again
```

Comment:
Aren't we already at this point past the level load?

Reply:
Thanks, fixed!